src/object/srv_obj.c

```diff
@@ -1456,7 +1456,7 @@ ds_obj_rw_handler(crt_rpc_t *rpc)
 	exec_arg.flags	  = flags;
 	/* Execute the operation on all targets */
 	rc = dtx_leader_exec_ops(&dlh, obj_tgt_update, &exec_arg);
-out:
```

Comment:
Directly move label "out" from line 1459 to line 1478, then do not need new label "dtx_end".